dist/generator-vulcanjs/lib/ui-text.js

```diff
@@ -2,6 +2,7 @@ const chalk = require('chalk');
 
 const descriptions = {
   appName: 'The name of your app',
+  doShallowClone: 'Default is fast and great for testing. Full vulcan git clone will avoid requiring to unshallow the repository to change the git remote.',
```

Comment:
I think this description creates more confusion than it solves. Why dont we just automate the `unshallow`?